build.gradle

```diff
@@ -840,6 +840,8 @@ project(':iceberg-arrow') {
       exclude group: 'org.codehaus.jackson'
     }
 
+    testImplementation 'org.apache.spark:spark-catalyst_2.12:3.5.1'
```

Comment:
I don't think we should be adding a spark dependency to the arrow module

Reply:
I have removed the spark dependency.